setup.py

```diff
@@ -17,6 +17,7 @@
 import setuptools
 
 REQUIRED_PACKAGES = [
+    'grpcio==1.7.3',
```

Comment:
nit: can you please reference https://issues.apache.org/jira/browse/BEAM-3357 in a comment?

Reply:
Done.